File: gump-engine-dev/gump-dev-web/src/main/antd/src/model/gumpMetadata.js
export default class GumpMetadata {

    constructor() {
        const urlParams = new URLSearchParams(window.location.search);
        this.appCode = urlParams.get('appCode');
    }

    get nodeId() {return this._nodeId;}
    set nodeId(nodeId) {this._nodeId = nodeId;}

    get code() {return this._code;}
    set code(code) {this._code = code;}

    get content() {return this._content;}
    set content(content) {this._content = content;}

    get desc() {return this._desc;}
    set desc(desc) {this._desc = desc;}

    get type() {return this._type;}
    set type(type) {this._type = type;}

}

File: gump-engine-runtime/gump-persistence/gump-persistence-core/src/main/java/org/junhui/gump/persistence/client/model/GumpMetadata.java
package org.junhui.gump.persistence.client.model;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class GumpMetadata {

    private final Long nodeId;

    private final String code;

    private final String desc;

    private final String type;

    private final String appCode;

    private final String content;

}
